src/go/k8s/pkg/console/sasl.go

```diff
@@ -270,3 +248,21 @@ func (k *KafkaACL) Cleanup(ctx context.Context) error {
 	controllerutil.RemoveFinalizer(k.consoleobj, ConsoleACLFinalizer)
 	return k.Update(ctx, k.consoleobj)
 }
+
+func (k *KafkaACL) createAdminClient(ctx context.Context) (KafkaAdminClient, *kadm.ACLBuilder, error) {
```

Comment:
Nit: this is not just creating the client, maybe we can just factor out the acl builder. the client builder is short enough to duplicate in both create/delete funcs

Reply:
The problem comes from the error context. I can change the name of the function, but I disagree to have same error messages across the code base.